File: JMoneroWallet/src/com/sylvyrfysh/monerowallet/renderers/Renderer.java
package com.sylvyrfysh.monerowallet.renderers;

import com.sylvyrfysh.monerowallet.WState;

import imgui.IO;
import imgui.ImGui;

public interface Renderer {
	public WState renderState();
	public void render(ImGui imgui,IO io);
}

File: JMoneroWallet/src/com/sylvyrfysh/monerowallet/renderers/Renderer.kt
package com.sylvyrfysh.monerowallet.renderers

import com.sylvyrfysh.monerowallet.WState
import imgui.IO
import imgui.ImGui

interface Renderer {
	fun renderState(): WState?
	fun render(imgui: ImGui?, io: IO?)
}